cdap-source-control/src/main/java/io/cdap/cdap/sourcecontrol/SourceControlConfig.java

```diff
@@ -37,10 +37,15 @@ public class SourceControlConfig {
 
   public SourceControlConfig(NamespaceId namespaceID, RepositoryConfig repositoryConfig,
       CConfiguration cConf) {
+    // if local dir is set use it for cloned storage
+    // for task workers it would use emptydir volume
+    String defaultCloneDir = cConf.get(Constants.SourceControlManagement.GIT_REPOSITORIES_CLONE_DIRECTORY_PATH);
+    String userDir = cConf.get(Constants.TaskWorker.WORK_DIR);
+    String gitCloneDirectory = defaultCloneDir != null ?
```

Comment:
Pls address the checkstyle warnings in the new code lines

Reply:
done